bundle/manifests/dell-csm-operator.clusterserviceversion.yaml

```diff
@@ -1852,4 +1852,4 @@ spec:
     name: Dell Technologies
     url: https://github.com/dell/csm-operator
   replaces: dell-csm-operator.v1.0.0
```

Comment:
Please modify replaces value with dell-csm-operator.v1.1.0 in all CSV's

Reply:
taken care